lib/user.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var UserStore = require('fh-wfm-user/lib/user/store');
```

Comment:
The store in -user is the simple-store with more methods around password checking added, pretty much findByUsername and manipulating passwords:

https://github.com/feedhenry-raincatcher/raincatcher-user/blob/feature/persistent-storage/lib/user/store.js#L7